third_party/istio-1.0.1/istio-lean.yaml

```diff
@@ -2879,14 +2877,14 @@ spec:
       serviceAccountName: istio-citadel-service-account
       containers:
         - name: citadel
-          image: "gcr.io/istio-release/citadel:release-1.0-20180823-09-15"
+          image: "docker.io/istio/citadel:1.0.1"
           imagePullPolicy: IfNotPresent
           args:
             - --append-dns-names=true
             - --grpc-port=8060
             - --grpc-hostname=citadel
             - --citadel-storage-namespace=istio-system
-            - --custom-dns-names=istio-pilot-service-account.istio-system:istio-pilot.istio-system
+            - --custom-dns-names=istio-pilot-service-account.istio-system:istio-pilot.istio-system,istio-ingressgateway-service-account.istio-system:istio-ingress.istio-system
```

Comment:
I'm not quite sure what this flag does, as it isn't often used in upstream Istio.  The help is for istio_ca is:
```
      --custom-dns-names string                    The list of account.namespace:customdns names, separated by comma.
```

This change doesn't look quite right.  ingress is deprecated in Istio 1.0.z.  Perhaps `istio-ingressgateway-service-account.istio-system:istio-ingressgateway.istio-system` is accurate.

Looking at the `ServiceAccount`:
```
apiVersion: v1
kind: ServiceAccount
metadata:
  name: istio-ingressgateway-service-account
  namespace: istio-system
  labels:
    app: ingressgateway
    chart: gateways-1.0.0
    heritage: Tiller
    release: RELEASE-NAME
```

we see the app label is ingressgateway, not ingress.

Cheers
-steve